### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by DarthGandalf**
Took it out of #159 because it's useful for non-printable build too

### CODE REVIEW

**Comment by qwrtln** [comment]
Thank you for this, let me check if it works and then merge it.

**Comment by qwrtln** [comment]
Can you run po4a here too?

**Comment by qwrtln** [comment]
Or not, wait until #167 is merged.

**Review by qwrtln — APPROVED** [review]

**Review by Heegu-sama — APPROVED** [review]

**Pull request merged**
